deepspeed/runtime/zero/stage2.py

```diff
@@ -132,6 +125,12 @@ def __init__(self,
                  gradient_predivide_factor=1.0,
                  gradient_accumulation_steps=1):
 
+        global flatten, unflatten
+        if flatten is None or unflatten is None:
+            utils_op = UtilsBuilder().load()
```

Comment:
Since this flatten/unflatten op logic is replicated in `engine.py` and `stage2.py`, maybe we should package it up and expose it as a util?